Sodium/Utils.swift

```diff
@@ -68,7 +68,7 @@ public class Utils {
 
         return hexData.withUnsafeMutableBytes { (hexPtr: UnsafeMutablePointer<Int8>) -> String? in
             bin.withUnsafeBytes { (binPtr: UnsafePointer<UInt8>) -> String? in
-                if sodium_bin2hex(hexPtr, hexDataLen, binPtr, bin.count) == nil {
+                guard nil != sodium_bin2hex(hexPtr, hexDataLen, binPtr, bin.count) else {
```

Comment:
Funnily enough this wasn't ordered this way to be a Yoda condition, but rather to place the smaller token `nil` at the beginning of the large conditional (as opposed to at the end where it could be read over more easily). I can appreciate this is a bit of a personal thing though – I'll swap this back later today :)